packages/geoprocessing-service/src/helpers/__tests__/copy-ntv2-format-files.spec.js

```diff
@@ -1,7 +1,5 @@
 import fs from 'fs'
 import path from 'path'
-import copyNtv2FormatFiles from '../copy-ntv2-format-files.cjs'
-
 console.log(process.cwd() + '**')
```

Comment:
@pwadmore-ea 
Is this console necessary now though?

Reply:
Refactored